public/jsx/views/complete-payment.jsx

```diff
@@ -18,6 +19,10 @@ module.exports = React.createClass({
     userEmail: React.PropTypes.string.isRequired,
   },
 
+  componentDidMount: function() {
+    tracking.setPage('/payment/complete');
```

Comment:
if nothing else, these should match the components/modules more closely. Maybe like: ``/complete-payment``, ``/card-listing``, ``/card-details``, and so on.